bddtests/steps/peer_basic_impl.py

```diff
@@ -231,6 +231,10 @@ def step_impl(context):
     assert context.transactionID != ""
     pass
 
+@when(u'I unconditionally query chaincode "{chaincodeName}" function name "{functionName}" on "{containerName}"')
```

Comment:
Does this look identical to the "I query chaincode" step definition or am I missing something? (And I guess you break it down to a separate step because you want to keep the semantics right?)